demisto_sdk/commands/validate/validate_manager.py

```diff
@@ -93,7 +93,7 @@ def __init__(self, is_backward_check=True, prev_ver=None, use_git=False, only_co
                                    FileType.DESCRIPTION,
                                    FileType.TEST_PLAYBOOK,
                                    FileType.TEST_SCRIPT,
-                                   )
```

Comment:
we also need to ignore this in RN check - look at `validate_no_missing_release_notes` - add it to the ignored files there